src/components/views/create/SuccessfulCreation.tsx

```diff
@@ -23,15 +25,15 @@ const SuccessfulCreation = (props: IProjectBySlug) => {
 				<GiverH4>High five!</GiverH4>
 				{/* <Image src='/images/motivation.svg' alt='motivation' width='121px' height='121px' /> */}
 				<SuccessMessage>
-					Your project is being reviewed by our team. You'll receive
-					an email from us once your project is listed.
+					Your project is being reviewed by our team. You&apos;ll
+					receive an email from us once your project is listed.
 				</SuccessMessage>
 				<SocialBox project={project} isSuccess />
 				<SuccessMessage>
 					You can still access your project from your account and
 					share it with your friends.
 				</SuccessMessage>
-				<Link passHref href='/projects'>
+				<Link passHref href={slugToProjectView(project.slug)}>
```

Comment:
@RamRamez 
why do you use slugToProjectView?

Reply:
since we may change the routing in future